config/environments/development.rb

```diff
@@ -71,6 +71,7 @@
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 
   config.hosts << ".circulate.local"
+  config.hosts << /[a-z0-9-.]+\.ngrok-free\.app/
```

Comment:
I use ngrok when testing this, and figure others might as well. Open to other services as well if they' assist with local dev.